src/operator/cudnn_convolution-inl.h

```diff
@@ -211,7 +211,7 @@ class CuDNNConvolutionOp : public Operator {
                filter_desc_,
                conv_desc_,
                out_desc_,
-               CUDNN_CONVOLUTION_FWD_PREFER_FASTEST,
+               CUDNN_CONVOLUTION_FWD_SPECIFY_WORKSPACE_LIMIT,
```

Comment:
@antinucleon I suspect this is what caused the NeuralStyle error when input is too large. Please check if this fixes it